src/json-validation.test.ts

```diff
@@ -0,0 +1,53 @@
+import anyTest, { TestInterface } from "ava";
+
+import { Instructions, RepoArray } from "./inputs";
+import { schemaNames, validateObject } from "./json-validation";
+
+const test = anyTest as TestInterface<{ db: string; tmpDir: string }>;
```

Comment:
I don't think we need any context here, so you can probably just import `anyTest` as `test` and skip this redefinition.

Reply:
Thanks for checking that. This line was copied from the other test but you're right we don't need it here.